viewing_party/main.py

```diff
@@ -0,0 +1,127 @@
+####################### Wave_01 #########################
+#Creat new movie with three attributes movie_title, genre and rating
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title and genre and rating:
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+#Add movie information to user watched list
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+#Add movie information to user watchlist  
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+#Add movie from watchlist to watched
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+    return user_data
+
+####################### Wave_02 #########################
+#Calculate average rating of movies in users watched list
+def get_watched_avg_rating(user_data):  
+    ratings = 0
+    cnt = len(user_data["watched"])
```

Comment:
When naming variables, I recommend opting for readability over conciseness.  In this case, calling the variable `count` makes everything that follows easier to read.